### DISCUSSION TITLE
404 File Not Found for Github Page (Netlify and local are ok)

### DISCUSSION THREAD

**Discussion by bakhrululum19** [jpanther/lynx#86] [Q&A]
Hi Guys,

I am using Lynx for my theme in Hugo, my repository is here: https://github.com/bakhrululum19/bakhrululum19.github.io

Locally I can access the pages I have built, and tried using Netlify: [bakhrululum19.netlify.app](https://bakhrululum19.netlify.app/) works well too.

The deployment status through both Actions and deploy from a branch always success. However the page [bakhrululum19.github.io](https://bakhrululum19.github.io/) still shows its not found 404.

Help me to solve this issue. Thanks.

Regards

**Comment by anarchivist** [comment]
You need to set up a Github Actions workflow to build the site. There's more detail about how to do this in the [Hugo documentation](https://gohugo.io/hosting-and-deployment/hosting-on-github/).
